### ISSUE TITLE
Use native sensu Ansible modules instead of templating

### ISSUE THREAD

**Issue by dmsimard** [centos-opstools/opstools-ansible#100]
Ansible has two relevant modules for Sensu: [sensu_check](https://docs.ansible.com/ansible/sensu_check_module.html) and [sensu_subscription](https://docs.ansible.com/ansible/sensu_subscription_module.html).

We should use them instead of trying to template one huge [checks.json file](https://github.com/centos-opstools/opstools-ansible/blob/master/roles/sensu/server/templates/sensu-checks.json.j2).

**Comment by mrunge** [comment]
right, that's what we should do. Somehow, this slipped during reviews.
